pkg/transport/generic_transport_controller.go

```diff
@@ -389,36 +389,22 @@ func (c *genericTransportController) getObjectsFromWDS(ctx context.Context, bind
 	objectsToPropagate := make([]*unstructured.Unstructured, 0)
 	// add cluster-scoped objects to the 'objectsToPropagate' slice
 	for _, clusterScopedObject := range binding.Spec.Workload.ClusterScope {
-		if clusterScopedObject.ObjectNames == nil {
-			continue // no objects from this gvr, skip
-		}
 		gvr := schema.GroupVersionResource{Group: clusterScopedObject.Group, Version: clusterScopedObject.Version, Resource: clusterScopedObject.Resource}
```

Comment:
FYI, here and elsewhere:

```suggestion
		gvr := schema.GroupVersionResource(clusterScopedObject.GroupVersionResource)
```

Reply:
fixed